connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -829,19 +837,16 @@ export async function garbageCollect({
   let stillAccessiblePagesCount = 0;
   let stillAccessibleDatabasesCount = 0;
 
-  // Find the resources not seen in the GC run
-  const resourcesToCheck = await findResourcesNotSeenInGarbageCollectionRun(
-    connector.id
-  );
-
-  for (const [i, x] of resourcesToCheck.entries()) {
+  let i = 0;
+  for (const x of batch) {
```

Comment:
nit: you can do `for const [i, x] of batch.entries()` to avoid ++ing your own `i`